src/components/UNSTABLE_Pagination/_mixins.scss

```diff
@@ -0,0 +1,152 @@
+////
+/// Unstable pagination mixins.
+/// @group unstable-pagination
+/// @copyright IBM Security 2019
+////
+
+@import 'carbon-components/scss/globals/scss/vars';
+
+@import '../../globals/layout/index';
+@import '../../globals/theme/index';
+@import '../../globals/type/index';
+
+@mixin unstable-pagination {
+  @include reset;
+  @include type-style('body-short-01');
+  width: 100%;
+  background-color: $ui-01;
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  border-top: 1px solid $ui-03;
+  height: rem(48px);
+  overflow: hidden;
+
+  &__text {
+    @include carbon--breakpoint('md') {
+      display: inline-block;
+    }
+
+    margin: 0 $carbon--spacing-05;
+    color: $text-02;
+  }
+
+  &__left,
+  &__right {
+    display: flex;
+    height: 100%;
+    align-items: center;
+
+    & > .#{$prefix}--form-item {
+      height: 100%;
+    }
+  }
+
+  &__left {
+    padding: 0 $carbon--spacing-05 0 0;
+  }
+
+  &__left &__text {
+    margin-right: rem(1px);
```

Comment:
What's the reasoning for using the `rem` method versus a `$carbon--spacing-*` variable or `carbon--mini-units` helper in places?

Reply:
Unfortunately there isn't a `$carbon--spacing-*` value that equals 1px, and using the `carbon--mini-units` helper would make this simple value MUCH harder to read 😅

Reply:
Good catch. I didn't see the `1px`! It looks like there's other hardcoded values that can benefit from this though? For example https://github.com/carbon-design-system/ibm-security/pull/249/files#diff-da6e5a620d3051e7e0e1e711c1862f87R54

Reply:
Also, there's a mix of the old spacing helpers and the newer spacing helpers too ( https://github.com/carbon-design-system/ibm-security/pull/249/files#diff-da6e5a620d3051e7e0e1e711c1862f87R54 ) - can we use the newer values so we won't have to worry about migrating them in future?

Reply:
@SimonFinney Unfortunately I can't tell where you are linking to? I try to go to those links & I get sent to the top of the "files" tab. I can't see any highlighted lines. Do you mind posting block code snippets? I could search for them then, if they are just in this file.

Reply:
😆 Oh, sorry! That's annoying - I could've sworn that worked. Lines 53 - 54:

```scss
&__right &__text {
    margin-right: 1rem;
```

And 124:

```scss
padding: 0 2.5rem 0 $spacing-md;
```

Reply:
Thank you! I can update those 👍